### ISSUE TITLE
PO007 gives warning if Quota named Q-EnforceQuota.xml

### ISSUE THREAD

**Issue by Kittiman** [apigee/apigeelint#223]
Hi,

I have a policy like this.

```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<Quota async="false" continueOnError="false" enabled="true" name="Q-EnforceQuota">
    <DisplayName>Q-EnforceQuota</DisplayName>
    <Properties/>
    <Identifier ref="client_id"/>
    <Allow countRef="apiproduct.developer.quota.limit" count="10000"/>
    <Interval ref="apiproduct.developer.quota.interval">1</Interval>
    <TimeUnit ref="apiproduct.developer.quota.timeunit">month</TimeUnit>
    <Distributed>true</Distributed>
    <Synchronous>true</Synchronous>
</Quota>

```
When running linter I get this warning.

warning: Non-standard prefix (null). Valid prefixes for Quota include: ["quota","q"] (PO007) at Source\gateway\apiproxy\policies\Q-EnforceQuota.xml:2:66

If I use QUOTA-EnforceQuota it works fine.

**Comment by DinoChiesa** [comment]
Thanks for reporting this. 
A pull request is now available.
see #224

**Comment by DinoChiesa** [comment]
The PR has been merged.

**Comment by ssvaidyanathan** [comment]
v2.2.0
